### PR TITLE
fix(ui): fix `videoCall` asset not available.

### PR DESCRIPTION
**Pull request by rlee1990**
Fixes: #1427 
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

**Comment by rlee1990** [comment]
@xsahil03x sorry did not see that one there sure thing. I will make that change now.

**Comment by rlee1990** [comment]
@xsahil03x done.

**Review by rlee1990 — COMMENTED** [review]
Removed the Jiffy update.

**Comment by xsahil03x** [comment]
@rlee1990, It still contains changes for `jiffy`.

**Review by rlee1990 — COMMENTED** [review]
Removed all Jiffy changes.

**Comment by xsahil03x** [comment]
@rlee1990 can you also add a changelog and run flutter format? Need to pass the checks.

File: packages/stream_chat_flutter/lib/src/localization/translations.dart [hunk 1]
```diff
@@ -83,7 +83,7 @@ abstract class Translations {
   /// in the [StreamMessageListView]
   String unreadMessagesSeparatorText(
     @Deprecated('unreadCount is not used anymore and will be removed ')
-        int unreadCount,
+    int unreadCount,
```
Comment (xsahil03x) [inline packages/stream_chat_flutter/lib/src/localization/translations.dart:86]:
```suggestion
        int unreadCount,
```

File: packages/stream_chat_flutter/lib/src/theme/message_theme.dart [hunk 2]
```diff
@@ -23,7 +23,7 @@ class StreamMessageThemeData with Diagnosticable {
     this.avatarTheme,
     this.createdAtStyle,
     @Deprecated('Use urlAttachmentBackgroundColor instead')
-        Color? linkBackgroundColor,
+    Color? linkBackgroundColor,
```
Comment (xsahil03x) [inline packages/stream_chat_flutter/lib/src/theme/message_theme.dart:26]:
```suggestion
        Color? linkBackgroundColor,
```

**Pull request merged**